tests/e2e_tests/cypress/integration/03-files.spec.js

```diff
@@ -66,7 +66,7 @@ context('Deployments', () => {
     // create an artifact to download first
     cy.get('.repository-list-item').contains('mender-demo-artifact').click().end()
     cy.get('a').contains('Create deployment').click({ force: true }).wait(5000).end()
-    cy.get('[placeholder="Select target group"]').click({ force: true })
+    cy.get('[placeholder="Select a device group to deploy to"]').click({ force: true })
     cy.get('[role="tooltip"]').get('li').contains('All devices').click().end()
     cy.get('button').contains('Create deployment').click().wait(30000).end()
```

Comment:
this will most likely also have to be adjusted to:
```
    cy.get('[role="tooltip"]').get('li').contains('All devices').click().wait(2000).end()
    cy.get('button').contains('Next').click().end()
    cy.get('button').contains('Create').click().wait(30000).end()
```
to work with the wizard behaviour... (the `.wait(2000)` should be enough to determine the device type compatibility)